index.js

```diff
@@ -20,7 +20,7 @@ module.exports = function(el) {
   var candidate, candidateIndex;
   for (var i = 0, l = candidates.length; i < l; i++) {
     candidate = candidates[i];
-    candidateIndex = candidate.tabIndex;
+    candidateIndex = parseInt(candidate.getAttribute('tabindex'), 10) || candidate.tabIndex;
```

Comment:
What if you set `focusable="false"` but not `tabindex`? I've read that `focusable="false"` is the only way to really prevent this behavior: https://stackoverflow.com/questions/18646111/disable-onfocus-event-for-svg-element.